assets/collage.css

```diff
@@ -154,9 +148,35 @@
   border: 0;
 }
 
-.collage-card .deferred-media__poster:focus,
-.collage-card .collage-card__link:focus {
+.collage-card .deferred-media__poster:after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  bottom: calc(var(--card-border-width) * -1);
+  left: calc(var(--card-border-width) * -1);
+  right: calc(var(--card-border-width) * -1);
+  top: calc(var(--card-border-width) * -1);
   outline-offset: 0.3rem;
+}
```

Comment:
I would update for video, and if there is a separate selector for card then, there as well if it's not too much effort. It's what we are doing for the pills selectors already. So it would be good to continue to do that with the outline approach. Even though it doesn't benefit safari users, the overall UX/UI feels better when the outline follows the border-radius of the card.